Felienne/hedy
# [UI FIX] Improves readability of achievements overlay

**Description**
In this PR we improve the readability of the achievements overlay by only making the background transparant instead of the whole element. Currently the overlay text is also partly transparant, making it harder to read. After this fix we only set this value to the background making the text still good readable.

**Fixes**
This PR fixes #3122.

**How to test**
Make sure you are logged in and navigate to the achievements page. It might be a bit hard to see, but the text should be better readable on the overlay.
